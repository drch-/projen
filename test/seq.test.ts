import { TestProject, synthSnapshot } from './util';

test('empty sequence', () => {
  const p = new TestProject();
  p.addSequence('empty');

  expect(synthSnapshot(p)).toMatchSnapshot();
});