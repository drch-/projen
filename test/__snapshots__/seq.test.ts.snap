// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty sequence 1`] = `
Object {
  ".gitignore": "# ~~ Generated by projen. To modify, edit .projenrc.js and run \\"npx projen\\".
!/.projen/sequences.json",
  ".projen/sequences.json": Object {
    "//": "~~ Generated by projen. To modify, edit .projenrc.js and run \\"npx projen\\".",
    "env": Object {},
    "seqs": Object {
      "empty": Object {
        "commands": Array [],
        "description": "empty",
        "env": Object {},
        "name": "empty",
      },
    },
  },
  "README.md": "# my project",
}
`;
